contents/handbook/strategy/overview.md

```diff
@@ -140,24 +142,37 @@ While there is value in the items on the right, we value the items on the left m
 
 * _Reject_: We enable our customers to ingest, store and analyze data on their infrastructure, we don't believe tons of integrations, dealing with multiple vendors and sending sensitive data to multiple cloud providers is the right approach. Providing all the apps and data platform in one place, removes the need to setup a warehouse in the first place.
 
-## Direction for 2022
-
- * 2 word summary: **Nail self-serve**
-    * **Customers**
-        * **Focus on high-growth startups.** E.g. Large initial contracts ($20k-$70k/year) and smaller deals in organizations that will eventually become large
-        * **Non Goal:** Start doing outbound sales
-    * **Product**
-        * **Goals:**
-            * **Quality:** Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
-            * **Self Service Subscription:** Customers of every size sign-up, start using and subscribe to PostHog without asking us for help
-        * **Non Goals:**
-            * Build lots of new low quality / partial features
-            * Build everything a single enterprise customer wants just to close a deal
-
-## Target customers for 2022
-
-We build for the highest-performing **product teams** (engineers, PMs, designers) building the **most loved products** at **high-growth startups**. We focus specifically on the  **product-engineers** (full-stack engineers skewed towards the frontend) but it should be usable by everyone in the product team.
+## Direction for 2023
+
+3-word summary: **Nail product engineering**
+
+* **Customers**
+  * **Focus on product engineers at high-growth startups.** E.g. Large initial contracts ($20k-$70k/year) and smaller startups that will eventually become large
```

Comment:
Just to clarify - is our definition now that 'product engineers at high-growth startups' includes both those at growth stage companies _and_ YC-type startups? Or are growth companies the primary target, and early stage startups secondary?

(I don't have a view on which is right, just that it would be good to clarify)

Reply:
Growth companies the primary target, and early stage startups secondary is the current framing. Building is much more focused on growth companies as primary target. I'd expect marketing to be more balanced.